cfgov/jinja2/v1/landing-page/index.html

```diff
@@ -14,129 +14,54 @@
-        'body':      'Heroes function as a primary focal point on a page,
-                      often used to introduce a collection of pages by
-                      combining a brief description of the goals of that
-                      section along with a visually impactful graphic.',
-        'image_url': 'http://www.consumerfinance.gov/know-before-you-owe/static/img/kbyo-hero_tablet-2x.png'
-    }, {
-        'background_color': '#d5e7e6',
-        'link_text':        'Hero call to action',
-        'link_url':         '#'
-    }) }}
+    {% for block in page.hero %}
+        {{ render_stream_child(block) }}
+    {% endfor %}
 {% endblock %}
 
-{# TODO: Remove lipsumText when and if it is no longer needed. #}
-{% set lipsumText = lipsum(1, false, 10, 25) | safe %}
-
 {% block content_main %}
-     <div class="block
+    <div class="block
                  block__flush-top">
         {# This molecule should always be first #}
-        {{ text_introduction.render({
-            'heading':      'Text Introduction',
-            'intro':        'Content Intro Lorem Ipsum',
-            'body':         lipsumText,
-            'link_text':    'Placeholder link',
-            'link_url':     '#'
-        }) }}
+        {% for block in page.introduction %}
+            {{ render_stream_child(block) }}
+        {% endfor %}
     </div>
     <div class="o-image-text-50-50-group">
-        {{ image_text_50_50.render({
-                'title' :       'Image and Text 50-50',
-                'image_path':   'http://placehold.it/1600x900',
-                'image_alt':    'This is a placeholder image',
-                'is_widescreen': true,
-                'description':  'When image conveys critical information
-                                and text is supporting it --
-                                i.e. chart, excerpt from document',
-                'is_button' :   true,
-                'link_url':     'http://www.google.com',
-                'link_text' :   'Visit Google'
-            }) }}
-            {{ image_text_50_50.render({
-                'title' :       'Image and Text 50-50',
-                'image_path':   'http://placehold.it/1600x900',
-                'image_alt':    'This is a placeholder image',
-                'is_widescreen': true,
-                'description':  'When image conveys critical information
-                                and text is supporting it --
-                                i.e. chart, excerpt from document',
-                'is_button' :   true,
-                'link_url':     'http://www.google.com',
-                'link_text' :   'Visit Google'
-            }) }}
+        {% for block in page.content %}
+            {% if block.block_type == 'image_text_50_50' %}
+                {{ render_stream_child(block) }}
+            {% endif %}
+        {% endfor %}
     </div>
     <div class="block">
         <h2>Headline for 2/7 organism</h2>
```

Comment:
Do you think it would be okay to make an atom molecule for a group title? Or should we adjust the image-text molecule to take this batching into account?